src/content/development/contribution-guide/_index.en.md

```diff
@@ -106,7 +106,21 @@ To be expanded
 
 #### General Availability
 
-To be expanded
+Once a release candidate is deemed stable and has no blocker bugs it will be released for general availability.
+Prior to releasing, the release manager will verify that there have been no changes on the stable branches since the last release
+candidate.
+This ensures that the release version is as stable as possible, and avoids any bugs that might have crept in and that could pose a risk
+to the stability of the released version.
+
+The new version will be announced per the [announcement guidelines] in the release process documentation.
+
+The [current release board](#current-release-board) will be closed, and all open tasks on it will be moved back to the
+[backlog board](#backlog-board).
+The tasks could then be considered for the next version, based on the [planning](#pre-planning) guidelines.
+
+During the week when the general availability release is performed, the next version will be [planned](#pre-planning).
+Additionally, a meeting to retrospect the last release cycle will be held.
```

Comment:
```suggestion
Additionally, a retrospective meeting for the last release cycle will be held.
```

Reply:
> I think L121 and L122 are not relevant here. All agile / scrum meetings are detailed in `Sprints` section of this guide, while this section belongs to the GA process

This is a general retrospective for the entire cycle, or do you think we dont need it?